### DISCUSSION TITLE
New condition for entry fields: only show entries viewable by the user

### DISCUSSION THREAD

**Discussion by darwinsoftware** [craftcms/cms#12240] [Ideas] [answered]
There is already a filter for entries that are editable by the user selecting (https://github.com/craftcms/cms/discussions/11995), but could we also add one for 'viewable' by the user selecting?

**Comment by brandonkelly** [comment, marked as answer]
Added for 4.4! (#12266)

Also renamed “Editable” to “Savable” to clear up a little ambiguity.

**Reply by darwinsoftware** [reply]
Thank you @brandonkelly

**Reply by brandonkelly** [reply]
Craft 4.4 is out!
